core/client/fs/src/main/java/alluxio/client/file/cache/evictor/UnevictableCacheEvictor.java

```diff
@@ -42,6 +44,12 @@ public PageId evict() {
     return null;
   }
 
+  @Nullable
+  @Override
+  public PageId evictMatching(Predicate<PageId> criterion) {
+    return null;
```

Comment:
non-blocking:  is it possible to use an optional?  we could change it in another pr

Reply:
Yes, will do in a separate PR once this merges.